src/lib.rs

```diff
@@ -133,7 +134,7 @@ impl Alpm {
   /// It behaves like `pacman -Q {query}`
   pub fn query_package_version<S>(&self, query: S) -> std::io::Result<String> where S: Into<String> {
     let s: String = query.into();
-    let mut cs = s.into_bytes();
+    let mut cs = s.as_bytes().to_vec();
```

Comment:
Since the query_package_version consumes the `query` anyways, this does not look that different to me.

As_bytes will create a cheap slice, but to_vec will copy the string.
Into_bytes will just free the String overhead, no copy. I prefere the into, because it copies less data.